setup.py

```diff
@@ -80,7 +87,15 @@
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
 
-    install_requires=install_requires,
+    #install_requires=install_requires,
+    install_requires=[
+        'pyodbc',
+        'six',
+        'sqlalchemy',
+        'geoalchemy>=0.7.3',
+        'shapely',
+        'pandas',
```

Comment:
If you are listing them here I recommend to get rid of the requirements files. There is a maintenance burden to keep both updated and they get easily out of sync.

(I like to use the req files and read them here though.)